packages/styles/src/variables/components/_badge.scss

```diff
@@ -12,7 +12,7 @@
 // Design System custom variables
 $badge-height: button.$btn-height-rg;
 $badge-gap: spacing.$size-regular;
-$badge-border-color: color.$gray-20;
+$badge-border-color: var(--post-gray-60);
```

Comment:
```suggestion
$badge-border-color: color.$gray-60;
```

Reply:
I am using CSS variables to make the badge display correctly on backgrounds.